.ci/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source /etc/os-release
+if [ "$ID" == fedora ];then
+	sudo -E dnf -y install automake  bats
+elif [ "$ID" == ubuntu ];then
+	sudo add-apt-repository -y ppa:duggan/bats
```

Comment:
Why is this needed? `bats` is in the Ubuntu repos so is it a minimum version dependency issue?